backend/PythonClient/multirotor/storage/gd_storage_service.py

```diff
@@ -18,6 +19,47 @@ def __init__(self, folder_id='1zZ06TaCzqdPJlQ9Uc4VgJaawBZm3eTSS'):
         """
         SCOPES = ['https://www.googleapis.com/auth/drive']
         self.credentials = service_account.Credentials.from_service_account_file(
-            'key.json', scopes=SCOPES)
+            'backend/key.json', scopes=SCOPES)
```

Comment:
The previous implementation was already trying to find `key.json` in the `backend` directory. Your implementation  causes an error: `FileNotFoundError: [Errno 2] No such file or directory: 'backend/key.json'`.